resource/traversers/dfu.cpp

```diff
@@ -217,6 +222,7 @@ int dfu_traverser_t::schedule (Jobspec::Jobspec &jobspec,
             break;
         }
         case match_op_t::MATCH_ALLOCATE:
+        case match_op_t::MATCH_GROW_ALLOCATION:
```

Comment:
I don't think I understand this syntax - does this mean do nothing for match allocate, or that match allocate is the same as match grow allocate? I would guess the latter since one line was just added (so we want EBUSY for both cases).

In that case, this is saying if no resources are available, just return busy. I think this answers my question about reserve - we can't reserve with an update. It's a yes / no, and right now, if we don't have it ask again later!

Reply:
> does this mean do nothing for match allocate

Kind of, yes. It's fall-through syntax, see: https://en.cppreference.com/w/cpp/language/switch.html or https://learn.microsoft.com/en-us/cpp/cpp/switch-statement-cpp?view=msvc-170

Reply:
Just to be extra clear, maybe add the standard fallthrough annotation here. That way if we turn on fallthrough warnings it will not come back and bug us later.

https://en.cppreference.com/w/cpp/language/attributes/fallthrough
```suggestion
        [[fallthrough]];
        case match_op_t::MATCH_GROW_ALLOCATION:
```